src/main/java/thaumicboots/events/BootsEventHandler.java

```diff
@@ -241,7 +213,7 @@ public void checkAir(EntityPlayer player) {
         Item item = player.inventory.armorItemInSlot(0).getItem();
 
         // meteor boots
-        if ((item instanceof ItemElectricMeteorBoots) || (item instanceof ItemMeteorVoidwalkerBoots)) {
+        if (item instanceof IMeteor && !(item instanceof IComet)) {
```

Comment:
`item instanceof IMeteor` should be enough to check for meteor based boots only. Rework your inheritance tree so it is possible pls. otherwise when you'll build up tech debt with supid long checks in the future (provided you'll add more boots)

Reply:
Have you looked at the code?

There's two boots that have two effects, that's what the last check is for

Reply:
if you make interfaces to label specific objects but then in the end you need to also check they are not labeled differently too, what's the point of labeling the specific objects in the first place?